hypertrace-graphql-labels-schema-impl/src/main/java/org/hypertrace/graphql/label/dao/LabelResponseConverter.java

```diff
@@ -60,5 +66,36 @@ private Label convertLabel(org.hypertrace.label.config.service.v1.Label label) {
     String key;
     String color;
     String description;
+    Map<String, List<LabeledEntity>> labeledEntitiesMap;
+
+    @Override
+    public LabeledEntityResultSet labeledEntities(String entityType, int limit) {
+      List<LabeledEntity> labeledEntities =
+          Optional.ofNullable(labeledEntitiesMap.get(entityType)).orElse(Collections.emptyList());
```

Comment:
If we're doing the work on the request side, this should be unreachable - can assume the map contains any requested entity types.

Reply:
Just to return empty values till implementation is done